openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -138,58 +138,53 @@
 - control_key: AU-5
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - key: a
-      text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-161
+      text: |      
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
+ 
```

Comment:
OpenShift uses its own logging mechanisms. Additional details in the control breakouts below.